### DISCUSSION TITLE
Which interpreter should it use? 3.9 or 3.10?

### DISCUSSION THREAD

**Discussion by zongbb1983** [pallets/flask#5416] [Q&A]
Which interpreter should it use? 3.9 or 3.10?  And what is its main project name? The "app.py"?  should I run? Thanks very much!

**Comment by davidism** [comment]
The latest available, unless you have a specific reason not to. You can find what the current version is at https://www.python.org/downloads/.

**Reply by zongbb1983** [reply]
Thanks for your reply. Well, my question is that now I used 3.9. When I run app.py, it shows like this:

Traceback (most recent call last):
  File "C:\Users\jack\PycharmProjects\flask-main\src\flask\app.py", line 6, in <module>
    import typing as t
  File "C:\Users\jack\PycharmProjects\flask-main\src\flask\typing.py", line 5, in <module>
    if t.TYPE_CHECKING:  # pragma: no cover
AttributeError: partially initialized module 'typing' has no attribute 'TYPE_CHECKING' (most likely due to a circular import)

what the reason is? Thanks very much! Does it concern to the interpreter?
